packages/playground-examples/copy/ja/3-7/Types and Code Flow/Uncalled Function Checks.ts

```diff
@@ -0,0 +1,43 @@
+//// { compiler: {  }, order: 1 }
+
+// 3.7では、関数の戻り値の代わりに誤って関数を
+// 使用してしまった場合に備えて、if文の中に
+// チェック機能が追加されました。
+
+// This only applies when the function is known
+// to exist making the if statement always be true.
+// これは関数が存在することがわかっていて、if文が常にtrueになる
+// 場合にのみ適用されます。
+
+// ここでは、オプションとオプションでないコールバックがある
+// プラグインインターフェースの例を示します。
+
+interface PluginSettings {
+  pluginShouldLoad?: () => void;
+  pluginIsActivated: () => void;
+}
+
+declare const plugin: PluginSettings;
+
+// pluginShouldLoadは存在しない可能性があるため
+// 次のチェックは正当なものとなります。
+
+if (plugin.pluginShouldLoad) {
+  // pluginShouldLoadが存在する場合のアクション
```

Comment:
「Do something」は「処理」と訳すのが一般的かと思っています
```suggestion
  // pluginShouldLoadが存在する場合の処理
```